### ISSUE TITLE
Visualizer Freezing Bug

### ISSUE THREAD

**Issue by KingSalty** [gethiox/GXAudioVisualisation#7]
Hello, I recently started using your audio visualizer and it's working great, but one exception. After rebaking the animation data about the fourth or fifth time, the literal bars will stand at their height stationary, even when the music plays. Rebaking again, recreating the visualizer base, reuploading the track, nothing works.
![screen shot 2016-03-27 at 4 38 58 pm](https://cloud.githubusercontent.com/assets/18104785/14067823/5c7271d4-f43c-11e5-8ec6-dc513a7d01c1.png)
In the picture, the bars remain stationary even when the track plays.
I have no idea what to do at this point because nothing will work, and I'll have to just start from scratch.
If you need more information just ask.

**Comment by gethiox** [comment]
Show me what is above of the visible part of the addon panel. I see soundfile input in right channel box, but maybe you has only selected 'left channel' from list.

**Comment by gethiox** [comment]
But if it work before corectly, that's weird.

**Comment by KingSalty** [comment]
I'm using the right channel.

**Comment by gethiox** [comment]
I will try to reproduce this problem later (in a 1-3 days), stay tuned and thanks to report issue.

**Comment by gethiox** [comment]
Also have you tried delete all of these objects? Emptys and cubes? Sometimes i had problem with that, when new emptys was created (with name like 'obj_r_1.001') and driver links was targeted to old emptys without any baked data.

**Comment by KingSalty** [comment]
Yes, like said on the wiki, if I had any problems I'd delete them and retry. So I did, including the emtpys, but resulted the same.

**Comment by gethiox** [comment]
Well, I can't reproduce problem, I playing with settings and baking many times (around 20), seems to working fo me, Can you specify when exactly plugin stops working? This bug can be triggered only by pressing "bake animation" many times?
Also I can propose another plugin that might be useful
https://youtu.be/D-y8REUdg6I
Here is possible to play around music visualisation without baking process, looks interesting but I don't test it.

**Comment by Alex9303** [comment]
I am having the same problem, except It does not work at all. [Here](https://github.com/gethiox/GXAudioVisualisation/files/493998/Audio.Visualization.Test.zip)
 are the files with the music.

**Comment by gethiox** [comment]
I will try to care about it in the next upcoming version that I am planing to write.
